elasticsearch.md

```diff
@@ -868,11 +959,17 @@ It handles LDAP authentication only using the configured LDAP connector (here `l
 ```yaml
 ldap_authorization:
   name: "ldap1"
-  groups: ["group3"]
+  user_belongs_to_groups:
```

Comment:
1. let's use simplified syntax 
2. I see we use deprecated `groups` syntax in our docs. Can we change it in all the docs to the new one?